source/lex.tex

```diff
@@ -288,6 +288,25 @@
 The \grammarterm{universal-character-name} construct provides a way to name
 other characters.
 
+\begin{bnf}
+\nontermdef{n-char}: one of\br
```

Comment:
`\nontermdef` already adds a colon. With the explicit additional colon, we now have two colons. This is not desired. Please remove the colons after `\nontermdef`.